### ISSUE TITLE
Keyboard navigation through tasks doesn't work correctly after invalid command

### ISSUE THREAD

**Issue by snss231** [snss231/ped#5]
![image.png](https://raw.githubusercontent.com/snss231/ped/main/files/be8677ed-829e-4e80-8aca-791643b60084.png)
